mewarpx/tests/test_utils_pulsing.py

```diff
@@ -0,0 +1,114 @@
+"""Test pseudo 1D diode run with the superLU solver."""
+import numpy as np
+from pywarpx import picmi
+
+from mewarpx.utils_store import util as mwxutil, pulsing
+
+def test_utils_pulsing_expr():
+
+    pulse_expr = pulsing.linear_pulse_function(
+        V_off=-1.0, V_on=7.5, pulse_period=20e-9, pulse_length=10e-9,
+        t_rise=5e-9, t_fall=5e-9
+    )
+    print(pulse_expr)
+    assert pulse_expr == "modT=(t-5e-09); p1=(modT>=0 and modT<5e-09); p2=(modT>=5e-09 and modT<(5e-09+1e-08)); p3=(modT>=(5e-09+1e-08) and modT<(5e-09+1e-08+5e-09)); p4=if(p1 or p2 or p3, 0, 1); dV=7.5--1.0; p1*(-1.0+dV/5e-09*modT)+p2*7.5+p3*(7.5-dV/5e-09*(modT-5e-09-1e-08))+p4*-1.0"
```

Comment:
I'm honestly not convinced by the utility of this as it mostly tests if it's implemented exactly the same, not that it's functionally the same. The test below seems to test functionality better. But I don't have a reason to remove it.

Reply:
I guess a reason to leave it is that if it passes and the one below doesn't, we know that something in getvoltage() or some other mewarpx/WarpX thing is causing that run to fail, not this expression creator. So I'm fine leaving it.